indigo_api/models/works.py

```diff
@@ -359,6 +362,55 @@ def as_at_date(self):
         if dates:
             return max(dates)
 
+    def all_provisions(self):
+        # a list of the element ids of the earliest PiT (in whichever language) of a work
+        # does not include Schedules as they don't have commencement dates
+        ids = []
+
+        def add_ids(toc):
+            for e in toc:
+                if e.id:
```

Comment:
Do we need to only include `section` or the tradition-specific equivalent here?

Reply:
I'd like to keep Chapters and Parts for now too – would be cool if we can use them in the form (e.g. select Chapter 2 and it automatically selects all the sections up to the start of Chapter 3)